README.md

```diff
@@ -1,2 +1,53 @@
-# app-elabftw
-electronic LabBook
+# ElabFTW BIBBOX application
+
+This container can be installed as [BIBBOX APP](https://bibbox.readthedocs.io/en/latest/ "BIBBOX") or standalone.
+ 
+After the installation follow these [instructions](INSTALL-APP.md)
+
+## Hints
+* approx. time with medium fast internet connection: **10 minutes**
+* This is a modification of the offical ELabFTW (https://www.elabftw.net/) applictaion. It uses **HTTP** instead of ~~HTTPS~~. Please keep this in mind when using this app.
+* Admin User: **set after instalations**
+* MySQL password **set during installations**
+
+
+## Install within BIBBOX
```

Comment:
Install within BIBBOX => Highlight that future steps are needed

Reply:
Further steps were expicitly mentioned in Standalone and BIBBOX install sections